projects/longest-word-in-string/index.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Find length of the longest word in Every Language
+layout: default
+date: 2020-10-25
+last-modified: 2020-10-25
+featured-image:
+tags: [longest-word-in-string, javascript]
+authors:
+    - barhouum7
+---
+
+In this article, we'll outline the longest word in a string project.
+
+## Requirements
+
+The purpose of this program is to return the length of the longest word in the provided sentence as follows:
+
+```javascript
+longest-word-in-string.lang "Google do a barrel roll"
```

Comment:
```suggestion
$ ./longest-word-in-string.lang "Google do a barrel roll"
```

Reply:
Why do you want to make this, like we want to run a bash shell script?

Reply:
This is just an example of how the file might be executed. It's not universal, but it shows how the program takes input from the command line.

Reply:
So you have to remove `./` if it is just an example of how a file of any language might be executed. Because `./` may refer of how executing a bash shell script.